src/org-manager/containers/organisation-list/organisation-list.component.ts

```diff
@@ -28,7 +29,8 @@ export abstract class OrganisationListComponent implements OnInit, OnDestroy {
     protected router: Router,
     protected ref: ChangeDetectorRef,
     protected organisationService: OrganisationService,
-    protected loadingService: LoadingService
+    protected loadingService: LoadingService,
+    protected sessionStorageService: SessionStorageService
   ) {
```

Comment:
all is fine , it may be better to adding providers on unit test those for session storage

Reply:
Fair point. My next ticket involves pagination which will need a significant amount of unit tests to be added, at which point I will add the SessionStorageService / a mock Session Storage to the test spec.